openedx_webhooks/tasks/pr_tracking.py

```diff
@@ -774,3 +776,8 @@ def update_labels_on_pull_request(self, *, labels: List[str]) -> None:
         logger.info(f"Patching labels on PR {self.prid}: {labels}")
         resp = get_github_session().patch(url, json={"labels": labels})
         log_check_response(resp)
+
+    def update_commit_status_for_cla(self, *, pull_request: PrDict) -> None:
+        result = update_commit_status_for_cla(pull_request)
+        if result is not None:
+            self.happened = True
```

Comment:
This doesn't match up with what `update_commit_status_for_cla` returns: it returns True or False for whether the cla is signed or not, even if it didn't have to change the status.

Reply:
Good catch. That bit evolved over time and you're correct; they no longer match. I have updated `update_commit_status_for_cla` to return the shape `tuple(has_changed: bool, has_signed: bool)`.